### ISSUE TITLE
Keep the page position fixed in presentation mode

### ISSUE THREAD

**Issue by us91** [xournalpp/xournalpp#4340]
**Is your feature request related to a problem? Please describe.**

When in presentation model, either full screen or within a window, the position of the page changes when moving back and forth across pages. To see this, just use View -> Presentation, and move the page down and then up, and observe the position of the page (the change is not much, but is obvious). 

**Describe the solution you'd like**
The position should be fixed. So that contents on different pages can have continuity when they are part of an animation, for example. Also the title line will be at the same position if it is a presentation.

**Describe alternatives you've considered**
There does not seem to be a workaround. 

**Additional context**
None

**Comment by rolandlo** [comment]
This is related to PR #2408, which is an unfinished attempt to create a nice presentation mode.

**Comment by forgottosave** [comment]
Since PR #2408 hasn't been worked on for over a year not and is outdated by almost 1000 commits I think we should star a new PR related to this issue.
 
I tracked the problem down to an update issue in the window height for the zoom, as well as some hard-coded margins for the presentation zoom. I'm on it to fix this.
Maybe in future I can work on a better presentation mode over all.

**Comment by ankit-ksh** [comment]
Yes I like xournal a lot, but this is the only feature that's very annoying about it. Presentation mode is a crucial part of this app since when we are annotating the presentation, we'd like the page to be fixed at one position. Now the page jumps up and down when we move forward or backward.
